src/modules/rest/Dto/UpdateFacilityControlDto.ts

```diff
@@ -43,5 +43,5 @@ export class UpdateFacilityControlDto {
     @IsObject()
     @IsOptional()
     @ApiModelProperty({example: null, default: null})
-    mapControl: MapControlEmbed;
+    mapControl: TerritoryControlMapControlEmbed;
```

Comment:
Thinking we should probably prefix this with metagame so it's clear is not OW related, like the rest are